sklearn/preprocessing/tests/test_label.py

```diff
@@ -203,8 +203,10 @@ def test_label_encoder_errors():
 
     # Fail on unseen labels
     le = LabelEncoder()
-    le.fit([1, 2, 3, 1, -1])
-    assert_raises(ValueError, le.inverse_transform, [-1])
+    le.fit([1, 2, 3, -1, 1])
+    msg = "contains previously unseen labels"
+    assert_raise_message(ValueError, msg, le.inverse_transform, [-2])
```

Comment:
I find the organization of the tests a bit weird but not your fault. The test that it actually works if they are present is way at the top of the file.

Reply:
Happy to reorganise tomorrow if you are able to give me some pointers - I'm not very familiar with the testing structure of sklearn as this is my first issue.

Reply:
it's fine, I think.